test/parser-recovery.js

```diff
@@ -288,11 +307,6 @@ describe('Invoke parser with incomplete expression', function() {
             var expr = jsonata('$equals3lucy[0].UnstructuredAnswers^()[0].Text', { recover: true });
```

Comment:
I can see where this expression has come from! While editing around here it might be good to remove the reference to the third-party company.

Reply:
fixed